week-3/todo-list/script.js

```diff
@@ -1,25 +1,108 @@
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
-}
+  list.innerHTML = "";
+
+  todos.forEach((item) => {
+    const listItem = document.createElement("li");
+    list.appendChild(listItem);
+    listItem.textContent = item.task;
+    // Create and append a checkbox
+    const checkBox = document.createElement("i");
+    listItem.appendChild(checkBox);
+    checkBox.classList.add("check-box");
+
+    const span = document.createElement("span");
+    listItem.appendChild(span);
+    span.classList.add("badge", "bg-primary", "rounded-pill");
+
+    // Create and append a check icon
+    // const checkIcon = document.createElement("i");
+    // checkIcon.classList.add("fa", "fa-check");
+    // span.appendChild(checkIcon);
 
+    const crashIcon = document.createElement("i");
```

Comment:
Is this a typo for "trash"or did you intend to call the variable as crashIcon?

Reply:
yes it is a typo ,I will fix it thanks for caring